c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -263,10 +263,20 @@ vars:
         # tinyows_host: {host}
 
     resourceproxy:
-        # list of urls from which it is safe to load content
-        targets:
+        # List of urls from which it is safe to load content
+        targets: {}
           #exempletargetname: http://www.camptocamp.com/?param1=%s&param2=%s
 
+        # List of allowed headers
+        allowed_headers:
+        - Cache-Control
+        - Content-Encoding
+        - Content-Language
+        - Content-Type
+        - Date
+        - etag
```

Comment:
I don't know if the case is important but it should be `ETag`

Reply:
+1, I will change that :-)